packages/vite-plugin-cloudflare/src/index.ts

```diff
@@ -467,6 +566,17 @@ export function cloudflare(pluginConfig: PluginConfig = {}): vite.Plugin[] {
 					})
 				);
 			},
+			async buildEnd() {
```

Comment:
This hook will also run at build time. I think that should be OK because everything inside will be a noop but wanted to check?

Reply:
Yep, `buildEnd` is more appropriate. I think it currently works fine as is but let's guard the logic to only apply in dev. It will be safer for future changes that way. I'll add a suggestion.

Reply:
what about `vite preview`. We'll want a similar cleanup mechanism there as well no?

Reply:
gotcha!